src/pages/Locations/LocationDetails/components/LocationPhotosGrid/LocationPhotosGrid.jsx

```diff
@@ -0,0 +1,197 @@
+import * as React from "react";
+import {useState} from "react";
+import {AppBar, Button, Dialog, Skeleton, Slide, Toolbar, Typography} from "@mui/material";
```

Comment:
Codacy has a fix for the issue: [Replace `AppBar,·Button,·Dialog,·Skeleton,·Slide,·Toolbar,·Typography` with `⏎··AppBar,⏎··Button,⏎··Dialog,⏎··Skeleton,⏎··Slide,⏎··Toolbar,⏎··Typography,⏎`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
import {
```